homeassistant/components/miele/strings.json

```diff
@@ -138,6 +138,17 @@
           "waiting_to_start": "Waiting to start"
         }
       }
+    },
+    "switch": {
+      "power_on": {
+        "name": "Power on"
+      },
```

Comment:
"Turn power on on" sounds strange, wouldn't it make more sense to set the `name` to `None`? as then we `turn the device on`?

Reply:
But given that there will be more entities, I think its logical that if we a switch doesn't start the whole machine for example, that's what you mean right?

Reply:
A matter of opinion. If we pull your suggestion further there may be many entities on the device page called "Washing machine".

Reply:
Just `Power` is a better name than `Power on`. However, on many appliances it is not controlling real power on/off but instead sort of a `wake up`. And the API will enable and disable it in somewhat different waiys in different models.
In the end the switch is needed for proper control of many devices. I suggest that we rename it to Power and go ahead with the PR.